meteor-dev/imports/ui/App.js

```diff
@@ -1,32 +1,78 @@
-import { Template } from 'meteor/templating';
-import './App.html';
-import { ClientsCollection } from '../../api/clients/clients';
+import { Template } from "meteor/templating"
+import { ReactiveVar } from 'meteor/reactive-var'
+import "./App.html"
+import { ClientsCollection } from "../../api/clients/clients"
+import { PlaybooksCollection } from "../../api/clients/playbooks.js"
+import { OutputLogsCollection } from "../../api/clients/outputLogs.js"
+import { AnsibleCollection } from "../../api/clients/ansible.js"
 
-Template.clientList.onCreated(function(){
-  Meteor.subscribe("Clients");
-});
+// Var that holds the currently selected log to display in the Output Log section
+var currentLogReactive = new ReactiveVar("")
+
+Template.clientList.onCreated(function () {
+  Meteor.subscribe("Clients")
+  Meteor.subscribe("Playbooks")
+  Meteor.subscribe("Ansible")
+  Meteor.subscribe("OutputLogs")
+})
 
 Template.clientList.helpers({
-  client: function(){
-    return ClientsCollection.find({}, {sort: { createdAt: -1}});
+  client: function () {
+    return ClientsCollection.find({}, { sort: { createdAt: -1 } })
+  },
+  option: function () {
+    return PlaybooksCollection.find({})
+  },
+  outputLog: function () {
+    return OutputLogsCollection.find({}, { sort: { timestamp: -1 } })
   },
-  provisionedClients: function(){
-    return ClientsCollection.find({provisioned: true}, {sort: { createdAt: -1}});
+  logText: function () {
+    return currentLogReactive.get()
   }
-});
+})
 
 Template.clientList.events({
-  "click .provision-button": function(event) {
-    event.preventDefault();
+  "click .provision-button": function (event) {
+    event.preventDefault()
+
+    //Talk to will about the meteor way of doing this:
+    const selectedOption = $(event.currentTarget)
```

Comment:
JQuery? Where is $ imported from?

Reply:
From what I see online it looks like jQuery is included globally by default in Meteor projects.